Tests/StoreKitUnitTests/ProductsManagerTests.swift

```diff
@@ -114,15 +114,20 @@ class ProductsManagerTests: StoreKitConfigTestCase {
     }
 
     fileprivate func createManager(storeKitVersion: StoreKitVersion,
+                                   storefront: StorefrontType? = nil,
                                    diagnosticsTracker: DiagnosticsTrackerType? = nil) -> ProductsManager {
         let platformInfo = Purchases.PlatformInfo(flavor: "xyz", version: "123")
+        let systemInfo = MockSystemInfo(
+            platformInfo: platformInfo,
+            finishTransactions: true,
+            storeKitVersion: storeKitVersion
+        )
+        if let storefront {
```

Comment:
I think this `if let` is not needed since `stubbedStorefront` is already optional